master/internal/core.go

```diff
@@ -1064,6 +1074,10 @@ func (m *Master) Run(ctx context.Context) error {
 	handler := m.system.AskAt(actor.Addr("proxy"), proxy.NewProxyHandler{ServiceID: "service"})
 	m.echo.Any("/proxy/:service/*", handler.Get().(echo.HandlerFunc))
 
+	m.echo.Any("/*", func (c echo.Context) error {
+		return echo.ErrNotFound
```

Comment:
wait this works? don't we still need to set it on the ctx?

Reply:
this works, but i'm not entirely sure why. i think this "catchall" returns to echo and some handler downstream sets the error on the context.

Reply:
yeah, i guess if a handler explicitly returns an error it is set somewhere in echo internally? but if no handler matches, the error doesn't actually get set.